server.js

```diff
@@ -0,0 +1,13 @@
+const server = require('express')();
+
+const dbRouter = require('./data/db-router.js');
+
+server.use(require('express').json());
```

Comment:
I mentioned this during standup last night, but because you explicitly have to reference this import again for the built-in middleware of express.json, you don't really save too much time establishing your server the way you do on line 1. it was more of a conceptual trick than one you should use all the time, but I am happy to see you implement just to prove to yourself that it does work :+1: